pkg/safecli/formatter.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package safecli
+
+import "strings"
+
+// ArgumentFormatter defines a function that formats a command argument to the string.
+type ArgumentFormatter func(a Argument) string
+
+// format combines arguments using ArgumentFormatter.
+func (f ArgumentFormatter) format(args []Argument) []string {
```

Comment:
I'm not saying that this is something unclear for me, but I'd prefer something more clear. Something like
```suggestion
func formatArguments(f ArgumentFormatter, args []Argument) []string {
```